src/libaktualizr/primary/CMakeLists.txt

```diff
@@ -1,23 +1,21 @@
 set(SOURCES aktualizr.cc
-            commands.cc
-            eventsinterpreter.cc
             initializer.cc
             reportqueue.cc
             sotauptaneclient.cc)
 
 set(HEADERS aktualizr.h
-            commands.h
-            eventsinterpreter.h
             initializer.h
             reportqueue.h
             sotauptaneclient.h)
 
 
 add_library(primary OBJECT ${SOURCES})
 
-add_aktualizr_test(NAME commands SOURCES commands_test.cc)
+# TODO: remove?:
```

Comment:
Can't we remove them already?

Reply:
Hahaha, yeah, but I want to keep the idea of some of those tests around still, but they have to be rewritten. I'm looking into that now. Could be a job for a follow-up PR.